org.opent2t.sample.thermostat.superpopular/org.opent2t.sample.thermostat.superpopular.json

```diff
@@ -49,7 +49,7 @@
         },
         "hvacMode": {
           "$ref": "oic.r.mode.json#/definitions/oic.r.mode",
-          "description": "Active & Supported heating/Cooling modes supported the device."
+          "description": "Active & Supported heating/Cooling modes supported the device. Should be one of ['coolOnly', 'heatOnly', 'auto', 'off']"
```

Comment:
I'm not sure if there's a better way of enforcing this in the schema without basically re-defining the entire oic.r.mode schema with the included restrictions.